tests/snapshots/tests.contrib.openai.test_openai.test_response_error.json

```diff
@@ -0,0 +1,37 @@
+[[
+  {
+    "name": "openai.request",
+    "service": "tests.contrib.openai",
+    "resource": "createResponse",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "",
+    "error": 1,
+    "meta": {
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "681e376800000000",
+      "component": "openai",
+      "error.message": "Error code: 400 - {'error': {'message': \"The requested model 'invalid-model' does not exist.\", 'type': 'invalid_request_error', 'param': 'model', 'code': 'model_not_found'}}",
+      "error.stack": "Traceback (most recent call last):\n  File \"/Users/xinyuan.guo/dd-repos/dd-trace-py/ddtrace/contrib/internal/openai/patch.py\", line 263, in patched_endpoint\n    resp = func(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/xinyuan.guo/dd-repos/dd-trace-py/.riot/venv_py31210_mock_pytest_pytest-mock_coverage_pytest-cov_opentracing_hypothesis6451_vcrpy421_urllib3~126_pytest-asyncio0211_pytest-randomly_openai1762_tiktoken_pillow/lib/python3.12/site-packages/openai/_utils/_utils.py\", line 287, in wrapper\n    return func(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/xinyuan.guo/dd-repos/dd-trace-py/.riot/venv_py31210_mock_pytest_pytest-mock_coverage_pytest-cov_opentracing_hypothesis6451_vcrpy421_urllib3~126_pytest-asyncio0211_pytest-randomly_openai1762_tiktoken_pillow/lib/python3.12/site-packages/openai/resources/responses/responses.py\", line 656, in create\n    return self._post(\n           ^^^^^^^^^^^\n  File \"/Users/xinyuan.guo/dd-repos/dd-trace-py/.riot/venv_py31210_mock_pytest_pytest-mock_coverage_pytest-cov_opentracing_hypothesis6451_vcrpy421_urllib3~126_pytest-asyncio0211_pytest-randomly_openai1762_tiktoken_pillow/lib/python3.12/site-packages/openai/_base_client.py\", line 1239, in post\n    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/xinyuan.guo/dd-repos/dd-trace-py/.riot/venv_py31210_mock_pytest_pytest-mock_coverage_pytest-cov_opentracing_hypothesis6451_vcrpy421_urllib3~126_pytest-asyncio0211_pytest-randomly_openai1762_tiktoken_pillow/lib/python3.12/site-packages/openai/_base_client.py\", line 1034, in request\n    raise self._make_status_error_from_response(err.response) from None\nopenai.BadRequestError: Error code: 400 - {'error': {'message': \"The requested model 'invalid-model' does not exist.\", 'type': 'invalid_request_error', 'param': 'model', 'code': 'model_not_found'}}\n",
+      "error.type": "openai.BadRequestError",
+      "language": "python",
+      "openai.base_url": "https://api.openai.com/v1/",
+      "openai.request.client": "OpenAI",
+      "openai.request.endpoint": "/v1/responses",
+      "openai.request.method": "POST",
+      "openai.request.model": "invalid-model",
+      "openai.request.user": "ddtrace-test",
+      "openai.user.api_key": "sk-...6KQA",
```

Comment:
Can we make sure these api key values in the snapshot tests are replaced with "sk-...key>"? Even if it's the last 4 characters let's make sure they're not exposed in our public repo